pam/secret_template.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: approval-pam-secrets
+metadata:
+  name: approval-pam-secrets
+objects:
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: approval-pam-secrets
+    labels:
+      app: approval
+  stringData:
+    backoffice-token: "${BACKOFFICE_TOKEN}"
+    backoffice-env: "${BACKOFFICE_ENV}"
+    approval-api-identity: "${APPROVAL_API_IDENTITY}"
+    kie-admin-user: adminUser
+    kie-admin-pwd: "${KIE_ADMIN_PWD}"
+    kie-server-user: executionUser
+    kie-server-pwd: "${KIE_SERVER_PWD}"
+parameters:
+- name: BACKOFFICE_TOKEN
+  displayName: Backoffice API token
+  required: true
+  description: Backoffice API token
+- name: BACKOFFICE_ENV
```

Comment:
Is this really a secret?

Reply:
not necessary. can be a plain parameter. I was thinking to place all related parameters in close locations.

The displayName and description need to be modified.